src/util-decode-mime.c

```diff
@@ -1609,6 +1600,16 @@ static int ProcessBodyLine(const uint8_t *buf, uint32_t len,
             remaining -= tobuf;
             offset += tobuf;
         }
+        if (ret == MIME_DEC_OK) {
+            ret = ProcessDecodedDataChunk(state->data_chunk, state->data_chunk_len, state);
```

Comment:
what does failure mean here for the memcpy below? Should we exit the function here?

Reply:
I think that previous failure should not prevent to save the end of line into the buffer for future addition

Failure of `ProcessDecodedDataChunk` is unimportant (just debug log)

Failure due to `DATA_CHUNK_SIZE - state->data_chunk_len` should be unreachable

Reply:
we do pass the error to the caller, right?

Reply:
And the callers just do debug logging about it, right ?